packages/lesswrong/components/votes/lwReactions/NamesAttachedReactionsVoteOnComment.tsx

```diff
@@ -1,9 +1,9 @@
 import React, { useState, useRef, RefObject, useEffect } from 'react';
 import { Components, registerComponent } from '../../../lib/vulcan-lib';
-import { CommentVotingComponentProps, } from '../../../lib/voting/votingSystems';
+import { CommentVotingComponentProps, NamesAttachedReactionsCommentBottomProps, } from '../../../lib/voting/votingSystems';
```

Comment:
still using `CommentVotingComponentProps`?

Reply:
Yes, for the top voting component